### DISCUSSION TITLE
Call to MapLibre Developers

### DISCUSSION THREAD

**Discussion by louwers** [maplibre/maplibre-native#898] [General]
To all developers in the MapLibre developer pool.

@Alistoteless @SnailBones @manhcuongincusar1 @orblivion @dineshdb @fynngodau @hami9x @HarelM @IvanSanchez @neocaos @jashanbhullar @jutaz @caspg @krishnaglodha @metasansana @pfertyk @subashtiwari1010 @tdurand @tempranova @Vincent-Cui @vinayakkulkarni @xiaomingfu 

We updated the [step-by-step guide](https://maplibre.org/roadmap/step-by-step-bounties-guide/) for working on bounties. It is now possible to suggest things you would like to work on as long as it fits within the [bounty directions](https://github.com/maplibre/maplibre/issues?q=is%3Aissue+is%3Aopen+label%3A%22bounty+direction%22) (even big things).

If you want to schedule a face-to-face to get to know each other, please feel free to schedule a call with me: https://calendly.com/bart-louwers/30min

I am very interested to learn about your ideas for the project as well as your motivation.

You can also reach out to @ovivoda (Ovi / Ovidiu Voda) or me (Bart Louwers) on [Slack](https://slack.openstreetmap.us/) any time.

Note: some developers may only be interested in contributing to MapLibre GL JS. If this is the case please let us know! :slightly_smiling_face:

**Comment by tdurand** [comment]
hey ! great that the bounty program is up and going ! on my end I got busy with some contract work so I don't have much time to dedicate to it... but I have it in mind when I have more buffer.. I would mostly be interested in improving the documentation and examples... as I am mostly familiar with maplibre gl js as an user and never really deep dived in the library implementation itself.

**Comment by IvanSanchez** [comment]
> some developers may only be interested in contributing to MapLibre GL JS

Yeah, that'd be me :grin: . It's not about a lack of interest, but rather my inability to do anything with c++/java. I'm way too deep in the dark passages of javascript, I guess.

**Reply by vinayakkulkarni** [reply]
Same! I'm deep into JavaScript world & trying to learn Rust but C++ / Java seems to be a place of unknowns & ChatGPT responses :P
